tools/llvm-spirv/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+option(LLVM_SPIRV_LIBSPIRV_DIS
+  "Use libspirv for disassembly." OFF)
```

Comment:
Perhaps it's worth clarifying that `libspirv` is from SPIRV-Tools.

Reply:
Yeah, done.